src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -349,11 +349,21 @@ runTestCase( const std::shared_ptr<xrt_core::device>& _dev, const std::string& p
 
   // Get out max thruput for bandwidth testcase
   if(xclbin.compare("bandwidth.xclbin") == 0) {
+    // old testcases where we have "Maximum throughput:"
     size_t st = os_stdout.str().find("Maximum");
     if (st != std::string::npos) {
       size_t end = os_stdout.str().find("\n", st);
       logger(_ptTest, "Details", os_stdout.str().substr(st, end - st));
     }
+    else {
+      // new test cases to find "Throughput (Type: {...}) (Bank count: {...}):"
+      auto st = os_stdout.str().find("Throughput", 0);
```

Comment:
Post 2021.2, we should find a better way to communicate this information back to xbutil.   "Screen Scraping" for the "Throughput" keyword of the console output is not the way it should be done.

Reply:
I agree